igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/InfoController.groovy

```diff
@@ -73,7 +73,24 @@ class InfoController {
             throw new MasterNotFoundException("Master '${master}' does not exist")
         }
 
-        jenkinsClient.jobs.list.collect { it.name }
+        def jobList = []
```

Comment:
would like to see a corresponding test in https://github.com/spinnaker/igor/blob/master/igor-web/src/test/groovy/com/netflix/spinnaker/igor/jenkins/InfoControllerSpec.groovy